### ISSUE TITLE
Make mnconvert work

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/mnconvert-ruby#1]
This is to implement https://github.com/metanorma/mnconvert/issues/16

This depends on https://github.com/metanorma/mnconvert/issues/15

**Comment by CAMOBAP** [comment]
@ronaldtse @Intelligent2013 as far as I see this task almost complete, I see several points to fix:
 - https://github.com/metanorma/mnconvert-ruby/blob/main/lib/mnconvert/version.rb#L2-L3 use only VERSION and the same version as in jar to automate releases
 - uncomment https://github.com/metanorma/mnconvert/blob/master/.github/workflows/release.yml#L82-L88

Let me know if I missed something?

**Comment by ronaldtse** [comment]
Sounds good, thanks! mnconvert is at version 1.8.0 so this matches.

**Comment by Intelligent2013** [comment]
Everything looks ok.

**Comment by CAMOBAP** [comment]
@Intelligent2013 may I ask you to have a look at https://github.com/metanorma/mnconvert/issues/16#issuecomment-875922711

And https://github.com/metanorma/mnconvert/pull/22 (low priority)

**Comment by CAMOBAP** [comment]
I didn't prepare PRs just pushed directly to `main` and added tests from mn2sts and sts2mn and they work

@ronaldtse @Intelligent2013 I think we are ready for release, manually this time

**Comment by CAMOBAP** [comment]
@Intelligent2013 let me know if there are some release planned for mnconvert soon, we will able to test autorelease https://github.com/metanorma/mnconvert/pull/23

**Comment by Intelligent2013** [comment]
> 
> 
> @Intelligent2013 let me know if there are some release planned for mnconvert soon, we will able to test autorelease [metanorma/mnconvert#23](https://github.com/metanorma/mnconvert/pull/23)

let's wait some days. I'm writing documentation (conversion rules) for mnconvert, after that I'll return to fixing mnconvert's issues.

**Comment by CAMOBAP** [comment]
@Intelligent2013 to be on the same page, do we have some critical issues right now? 

I'm asking because all tests are green, and probably I can do 1.8.0 release just to unblock @opoudjis for https://github.com/metanorma/metanorma-standoc/issues/492

@Intelligent2013 how do you think?

**Comment by Intelligent2013** [comment]
@CAMOBAP ah ok, no problem. It can be released. There aren't any critical issue. Please do it.

**Comment by CAMOBAP** [comment]
Released https://rubygems.org/gems/mnconvert
